composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "laravelcollective/html",
+  "name": "andrey-helldar/html",
```

Comment:
OMG. I use this module in your project by connecting it through the composer. If you leave "laravelcollective / html", an error occurs. That renamed.
Before you pull request forgot to rename to the correct name.